core/controllers/pages_test.py

```diff
@@ -15,14 +15,149 @@
+        self.assertEqual(response.content_type, 'text/html')
+
+
+class GetStartedPageTest(test_utils.GenericTestBase):
+    """Test for get started page."""
+
+    def test_loading_get_started_page_loads_correctly(self):
+        response = self.testapp.get('/get_started')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+        response.mustcontain('I18N_GET_STARTED_PAGE_HEADING')
+
+
+class TeachPageTest(test_utils.GenericTestBase):
+    """Test for teach page."""
+
+    def test_loading_teach_page_loads_correctly(self):
+        response = self.testapp.get('/teach')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+        response.mustcontain('I18N_TEACH_PAGE_HEADING')
+
+
+class ContactPageTest(test_utils.GenericTestBase):
+    """Test for contact page."""
+
+    def test_loading_contact_page_loads_correctly(self):
+        response = self.testapp.get('/contact')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+        response.mustcontain('I18N_CONTACT_PAGE_HEADING')
+
+
+class DonatePageTest(test_utils.GenericTestBase):
+    """Test for donate page."""
+
+    def test_loading_donate_page_loads_correctly(self):
+        response = self.testapp.get('/donate')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+
+
+class ThanksPageTest(test_utils.GenericTestBase):
+    """Test for thanks page."""
+
+    def test_loading_thanks_page_loads_correctly(self):
+        response = self.testapp.get('/thanks')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+
+
+class ForumPageTest(test_utils.GenericTestBase):
+    """Test for forum page."""
+
+    def test_loading_forum_page_loads_correctly(self):
+        response = self.testapp.get('/forum')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+
+
+class TermsPageTest(test_utils.GenericTestBase):
+    """Test for terms page."""
+
+    def test_loading_terms_page_loads_correctly(self):
+        response = self.testapp.get('/terms')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+
+
+
+class PrivacyPageTest(test_utils.GenericTestBase):
+    """Test for privacy page."""
+
+    def test_loading_privacy_page_loads_correctly(self):
+        response = self.testapp.get('/privacy')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+
+
+class AboutRedirectPageTest(test_utils.GenericTestBase):
+    """Test for about redirect page."""
+
+    def test_redirect_to_about_page_loads_correctly(self):
+        response = self.testapp.get('/credits')
+        self.assertEqual(response.status_int, 302)
```

Comment:
Check the destination page URI too. Ditto below for all the "redirect pages".

Reply:
> Check the destination page URI too. Ditto below for all the "redirect pages".

suggest me, here i have to test about page with url="/about" or any thing else please tell me.

Reply:
See my other comment.

Reply:
@netajik, is this comment resolved?